null.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var RingBuffer = require('ringbufferjs');
+
+module.exports = NullStatsd;
+
+function NullStatsd(capacity) {
```

Comment:
Might be out of scope but is it worth making these functions spies (using say Sinon) so that they can be easily inspected during testing. Although thanking while I type that might be better as an actual separate mock/fake.